assets/project/attosoc.yaml

```diff
@@ -24,3 +24,5 @@ required_toolchains:
   - vpr-fasm2bels
   - nextpnr-xilinx
   - nextpnr-fpga-interchange
+skip_toolchains:
```

Comment:
Please, keep the last line. See other YAMLs below as well.

Reply:
Done.